backend/src/services/emailService.ts

```
@@ -20,54 +21,99 @@ export class EmailService {
+      this.transporter = null;
+    }
+
+    return this.transporter;
+  }
+
+  private static isValidEmail(email: string): boolean {
+    return /^[^\s@]+@[^\s@]+\.[^\s@]+$/.test(email);
+  }
+
+  private static async sendEmail(options: EmailOptions): Promise<boolean> {
+    if (!this.isValidEmail(options.to)) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)